src/Components/Artwork/Details/BidTimerLine.tsx

```diff
@@ -13,22 +15,30 @@ interface BidTimerLineProps {
 export const BidTimerLine: React.FC<BidTimerLineProps> = ({ artwork }) => {
   const data = useFragment(bidTimerLineFragment, artwork)
 
-  const { collectorSignals } = data
+  const { collectorSignals, saleArtwork } = data
   const { lotClosesAt, registrationEndsAt, onlineBiddingExtended } =
     collectorSignals?.auction ?? {}
-  const { time } = useTimer(lotClosesAt ?? "")
+
+  const [updatedLotClosesAt, setUpdatedLotClosesAt] = useState(lotClosesAt)
+
+  useAuctionWebsocket({
+    lotID: saleArtwork?.lotID!,
+    onChange: ({ extended_bidding_end_at }) => {
+      setUpdatedLotClosesAt(extended_bidding_end_at)
+    },
+  })
+
+  const { time, hasEnded: hasBiddingEnded } = useTimer(updatedLotClosesAt!)
   const { days, hours, minutes, seconds } = time
   const { isAuctionArtwork } = useArtworkGridContext()
-  const biddingEnded = lotClosesAt && new Date(lotClosesAt) <= new Date()
-  const registrationEnded =
-    registrationEndsAt && new Date(registrationEndsAt) <= new Date()
+  const { hasEnded: hasRegistrationEnded } = useTimer(registrationEndsAt!)
```

Comment:
Similar for bidding registration period, use the hook to avoid a `new Date()` check.